index.html

```diff
@@ -4,21 +4,48 @@
     	<meta charset="utf-8">
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one,two,three">
+		
 
 		<title>Blank Template</title>
-
+		
 		<!-- CSS links -->
+		<link rel="stylesheet" type="text/css" href="css/normalize.css">
+		<link rel="stylesheet" type="text/css" href="css/style.css">
 
 	</head>
 	<body>
-		<div>
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
-			</header>
-			<footer>
-				&copy; Footer Goes Here
-			</footer>
-		</div>
+		<header>
+			<div class="container">
+				<h1>Relaxr</h1>
+				<h2 class="text-header">Get piece of mind with a single tap</h2>
+				<p class="button-header-background">
+					<button class="button-header">Get It Now</button>
+				</p>
+			</div>
+		</header>
+		<section>
+			<div class="container">
+				<h2 class="subtitle-section">Benefits</h2>
+				<h3 class="text-section">The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</h3>
+				<ul>
+						<li>Schedule meetings for you</li>
+						<li>Excel automation to complete work for you</li>
+						<li>Responds to emails on your behalf</li>
+						<li>Does all your work for you with our revolutionary Al technology</li>
+				</ul>
+			</div>
+		</section>
+		<footer>
+			<div class="container">
+				<blockquote>
+					<p>"Relaxr changed my life. I've been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work." 	 <span>-Amanda Inuit</span></p>
```

Comment:
You could use a `<cite>` tag here instead of a `<span>`; it's slightly more semantically interesting.